### ISSUE TITLE
Dragon's Breath error on use

### ISSUE THREAD

**Issue by Iiridayn** [TheUnderTaker11/GeneticsReborn#54]
When I press the keybind for Dragon's Breath (I have the gene, added through normal survival mechanics - also impacts all other players on the server with the gene), nothing happens in the client, and the server reports:

```
[12:24:23] [Server thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NoSuchMethodError: net.minecraft.entity.projectile.Enti
tyDragonFireball: method <init>(Lnet/minecraft/world/World;DDDDDD)V not found
        at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_144]
        at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_144]
        at net.minecraft.util.Util.func_181617_a(SourceFile:46) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:668) [MinecraftServer.cl
ass:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387) [ld.class
:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.cl
ass:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_144]
Caused by: java.lang.NoSuchMethodError: net.minecraft.entity.projectile.EntityDragonFireball: method <init
>(Lnet/minecraft/world/World;DDDDDD)V not found
        at com.theundertaker11.geneticsreborn.packets.SendShootDragonBreath$Handler2$1.run(SendShootDragon
Breath.java:64) ~[SendShootDragonBreath$Handler2$1.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_144]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_144]
        at net.minecraft.util.Util.func_181617_a(SourceFile:45) ~[h.class:?]
        ... 5 more
```

**Comment by Iiridayn** [comment]
Ah - server is running 1.10.2, latest release. Also looking forward to 1.12.2 to come out of beta - as this isn't a critical bug, fixing in the 1.12.2 release is fine for me :).

**Comment by Iiridayn** [comment]
Ah, might be a duplicate of #39, but with details.

**Comment by alothmanmoe** [comment]
Since i have updated to 1.12-1.12.2 i am no longer supporting 1.10.2, that being said, i will test with the issue on 1.12 as well and fix it if it is persisting

**Comment by Iiridayn** [comment]
On Curse those versions are listed as Beta - are they considered stable then? Thanks.

**Comment by alothmanmoe** [comment]
Alpha - Unstable, crashes, bugs
Beta - Playable, functional, minimal bugs
Release - Playable, fully functional, minimal-no bugs
that's my rating scale, and what i'm using for uploaded versions

**Comment by TheUnderTaker11** [comment]
Fixed for both 1.10.2 and 1.11.2. Problem probably persist for 1.12 so will leave this thread here.

**Comment by alothmanmoe** [comment]
This issue has been fixed for 1.12, i will post a release later today after some more changes
